.prettierignore

```diff
@@ -0,0 +1 @@
+CHANGELOG.md
```

Comment:
Not sure why this is here now. Should this go to a separate mini PR so it gets merged and applies to all PRs @mattseddon ?

Reply:
I had to add it for #1883. I can move it out if you want but it is fine for it to be here given how much time we have left.